client/status/container.go

```diff
@@ -5,6 +5,11 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/status"
 )
 
+var (
+	ErrEACLNotFound      = EACLNotFound{}
```

Comment:
docs required. Lets mention that these values must not be mutated and should be used for `errors.Is` calls only

Reply:
done